### DISCUSSION TITLE
Chrome Extension

### DISCUSSION THREAD

**Discussion by FormalSnake** [excalith/excalith-start-page#39] [Ideas]
### Please Describe The Feature You Would Like To See
To have this as a chrome extension, I can help.

**Comment by excalith** [comment]
Hello @FormalSnake, thanks for the request, I have converted a discussion topic from that.

I was thinking about making browser extensions, however due to the rapid updates with huge refactors thanks to great ideas from community, it is better to stick with the one where you control the updates such as fork or docker. This will prevent data mismatching latest update issues.

However, I would like to see this as an extension in the future, probably after the data structure becomes more solid without the need for refactoring.

I would really love to hear your opinion regarding this.
Cheers!

**Reply by FormalSnake** [reply]
I do not see a problem personally, all that is necessary is a manifest.json. the only issue with redactors would be localstorage for the extension. I made a new tab extension called Ultimate new tab, which used react.

**Reply by excalith** [reply]
Hmm, so we should use the counterpart for localstorage for extensions. This could be done through github actions where we force the build to use storage coutnerpart for extensions. 

My only concern is the constant updates that can break user configurations, which would break the app. Instead of resolving issues by manually migrating into new data, it is better to have a clone of the project or the docker image with a spesific image version where you have the total control over the versions. 

I believe one day we will have a solid structure and will be easy to deploy pretty much anywhere, including using the currently preview build on the web viable.

**Reply by FormalSnake** [reply]
That could be a concern, but by how chrome storage works is storing data under a name, which could easily be converted. But it does become a problem when it is from X updates ago
